### PR TITLE
updating param tests and adding domain parsing tests

### PR DESCRIPTION
**Pull request by ericfichtel**
Creating this PR to update the mbox param tests that we updated in the dot notation fix across the SDKs. This PR also includes domain parsing tests for the new PSL domain parsing functionality. 

## Related Issue
Param tests:
https://jira.corp.adobe.com/browse/TNT-45620, https://jira.corp.adobe.com/browse/TNT-45549, https://jira.corp.adobe.com/browse/TNT-45619

Domain Parse tests:
https://jira.corp.adobe.com/browse/TNT-40238
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by jasonwaters — CHANGES_REQUESTED** [review]
Eric,

These artifacts come from [real activities](https://github.com/adobe/target-sdk-testing#test-artifact-data) created in the `targettesting` tenant.  

So what you need to do is: 

1. [create this activity in the target admin](https://experience.adobe.com/#/@targettesting/target/activities) ( i have granted you access, check your email)
2. Update`testArtifacts.json` to include the ID of the activity you created in the `activityIds` array of the `artifactFilename` you want it to appear in.  In your case it looks like `TEST_ARTIFACT_PARAMS.json`.
3. [Generate updated test artifacts](https://github.com/adobe/target-sdk-testing#generating-updated-test-artifacts). This will actually download the latest artifact and generate the test artifacts based on the json file from step 3.
4. Update the [On-device decisioning test artifact wiki](https://wiki.corp.adobe.com/display/Target/On-device+Decisioning%3A+Test+Artifacts) to include the activity you created in the appropriate table with details. 

You'll want to follow this same procedure to update the tests for the domain parsing work you did as well.  Might as well include it as part of this PR?

**Comment by ericfichtel** [comment]
Hi Jason, if the activities I created look good I will update the wiki. I put these new artifacts into the Node SDK.

**Review by jasonwaters — COMMENTED** [review]
LGTM @ericfichtel .  

Will you also please update the README.md file with any details that were missing such as instructions to update the test suite?

File: README.md [hunk 1]
```diff
@@ -97,7 +106,7 @@ The test artifacts are generated from a real production artifact.  This artifact
 | organizationId | 65453EA95A70434F0A495D34@AdobeOrg |
```
Comment (ericfichtel) [inline README.md]: do we want to update these with the target testing organization instead?

File: README.md [hunk 2]
```diff
@@ -58,6 +58,15 @@ For each test suite file...
 
 To see a working example check out the [Node.js test runner](https://github.com/adobe/target-nodejs-sdk/blob/main/packages/target-decisioning-engine/test/decisioning.spec.js)
 
+### Adding Additional Tests
+
+To add additional tests to the suite:
+1. Create a new activity in the `Target Testing` admin [here](https://experience.adobe.com/#/@targettesting/target/activities)
+1. Find the object that contains the `artifactFilename` you wish to update in `src/testArtifacts.json`. Update `activityIds` and `adminUrls` with the activity ID and admin URL of the new activity
+1. Follow the steps [below](#generating-updated-test-artifacts) to update the artifacts located at `src/schema.artifacts` with the new activity
+1. Update the corresponding test model located at `src/schema/models` with the new activity. Specifications for the test models can be found [here](#test-models)
```
Comment (jasonwaters) [inline README.md]: please mention on step 4 that this should be done by using the `updateTestSchema.sh` script mentioned in  #updating-to-the-latest-schema

File: README.md (hunk 1 above)
Comment (jasonwaters) [reply README.md]: Yes good catch!  please update them

**Review by jasonwaters — APPROVED** [review]
looks good! just a few comments

**Review by dcottingham — APPROVED** [review]
Thanks for adding the instructions to the README

**Pull request merged**